### ISSUE TITLE
Training model builder error for Forecasting due to datetime

### ISSUE THREAD

**Issue by vzhuqin** [dotnet/machinelearning-modelbuilder#2022]
**System Information (please complete the following information):**
 - Model Builder Version (available in Manage Extensions dialog): 16.13.2.2211002
- Microsoft Visual Studio Enterprise 2022 (64-bit) Version 17.0.6

**Describe the bug**
- On which step of the process did you run into an issue: Forecasting>Training error

**Data source**
https://github.com/dotnet/machinelearning-samples/raw/main/samples/csharp/getting-started/Forecasting_BikeSharingDemand/BikeDemandForecasting/Data/DailyDemand.mdf

**To Reproduce**

1. Select Create a new project from the Visual Studio 2022 start window;
2. Choose the C# Console App (.NET Core) project template with .Net 6.0;
3. Add model builder by right click on the project;
4. Select Forecasting and input above data source;
5. Set "TotalRentals" as Label, "Rental Data" as Time series;
6. Start training, prompts model builder error that: Unable to cast object of type 'System.DateTime' to type 'System.String

**Expected behavior**
No error during training.

**Screenshots**
Data settings:
![image](https://user-images.githubusercontent.com/81727020/153532593-fd6c579d-8155-4637-925f-ffe303d9858b.png)

Error screenshot:
![image](https://user-images.githubusercontent.com/81727020/153532468-f00b4e64-46f2-4672-ad64-910682c84a06.png)

**Additional context**
Error message:
   at System.Data.SqlClient.SqlBuffer.get_String()
   at Microsoft.ML.Data.DatabaseLoader.Cursor.<>c__DisplayClass38_0.<CreateStringGetterDelegate>b__0(ReadOnlyMemory`1& value)
   at Microsoft.Data.Analysis.StringDataFrameColumn.AddValueUsingCursor(DataViewRowCursor cursor, Delegate getter)
   at Microsoft.ML.IDataViewExtensions.ToDataFrame(IDataView dataView, Int64 maxRows, String[] selectColumns)
   at Microsoft.ML.ModelBuilder.AutoMLService.Proposer.ResampleStrategyProposer.InitializeForecasting(TrialParameter trialParameter) in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/Proposer/ResampleStrategyProposer.cs:line 127
   at Microsoft.ML.ModelBuilder.AutoMLService.Proposer.ResampleStrategyProposer.Propose(TrialParameter trialParameter) in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/Proposer/ResampleStrategyProposer.cs:line 51
   at Microsoft.ML.ModelBuilder.AutoMLService.Proposer.Controller.Propose() in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/Proposer/Controller.cs:line 66
   at Microsoft.ML.ModelBuilder.AutoMLService.Experiments.LocalAutoMLExperiment.<ExecuteAsync>d__15.MoveNext() in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/Experiments/LocalAutoMLExperiment.cs:line 163
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.ML.ModelBuilder.AutoMLEngine.<StartTrainingAsync>d__21.MoveNext() in /_/src/Microsoft.ML.ModelBuilder.AutoMLService/AutoMLEngineService/AutoMLEngine.cs:line 160

**Comment by cmckechney** [comment]
I am having the exact same error.  ML 2022 and VS2022 w/ SQL Server data column

**Comment by beccamc** [comment]
Just checked this with the addition of DateTime type and it looks like it's blocked by the DataFrame bug - https://github.com/dotnet/machinelearning/issues/6213

**Comment by beccamc** [comment]
Duplicate of https://github.com/dotnet/machinelearning-modelbuilder/issues/2166. This fix will ship in the Fall Model Builder release.
